boards/arm/quick_feather/Kconfig.defconfig

```diff
@@ -11,7 +11,7 @@ config BOARD
 if PWM
 
 config PWM_LITEX
-	select SOC_EOS_S3_FPGA
+	select EOS_S3_PROGRAM_FPGA
```

Comment:
The main change I made in these commits is to program FPGA from M4 program itself, instead of doing it outside using Jlink. To enable this feature I have added EOS_S3_PROGRAM_FPGA. If someone wants to use the old way, one can enable  SOC_EOS_S3_FPGA. I have not made any changes in this flow. Since all clocks are enabled with EOS_S3_PROGRAM_FPGA, other is not required. So EOS_S3_PROGRAM_FPGA and  SOC_EOS_S3_FPGA are mutually exclusive.

Reply:
Does `eos_s3_pwm_ip.h` contain `PWM_LITEX`? If not you must add separate config entry like explained [here](https://github.com/antmicro/zephyr/pull/3/files#r444115629).

Reply:
No it doesn't.
I'll add separate config entry.

Reply:
Added new configuration. However I understand from our H/W team that we are using litex pwm IP only, the bit stream is generated using symbi flow tools (with EOS S3 support). Our driver just loads the FPGA IP. It doesn't have full functions. For that I have to use pwm litex driver. So I have included that too.